### PR TITLE
Update docs to reference RocketMap and fix install links

### PR DESCRIPTION
**Pull request by BrownSlaughter**
## Description
Update docs to reference RocketMap and fix install links

## Motivation and Context

Project rename meant links and references were outdated

## Types of changes

- [x] Document Change

## Checklist:

- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/extras/faq.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Common Questions and Answers
 
+## Critical Error "Missing sprite files"
+Sprite files are the files that contain all the pokemon icons, Unfortunatly due to the need to protect from copywrite situation, sprite files can no longer be part of the project. These files can probably be located from other users in the [RocketMap Discord](https://discord.gg/PWp2bAm). While you wait feel free to ask for a drink with the !sprite command.
```
Comment (onilton) [inline docs/extras/faq.md]: @sebastienvercammen @FrostTheFox  is this a good idea? Mentioning how to get the sprites here?

Comment (sebastienvercammen) [inline docs/extras/faq.md]: Nope, needs to be removed.

Comment (BrownSlaughter) [inline docs/extras/faq.md]: it doesn't actually tell you how to get the sprites apart from ask other users, cant see what the problem would be. is it just the drink part to be removed or asking in discord as well?

Comment (divnotded) [inline docs/extras/faq.md]: I know the project is world wide but  here is why even mentioning how to go somewhere else is a bad idea in the US MGM Studios, Inc. v. Grokster Ltd., 125 S. Ct. 2764 (2005) this was a unanimous decision about secondary liability by SCOTUS

**Pull request merged**